src/main/java/core/basesyntax/service/ConvertWriteDataService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.Map;
+
+public interface ConvertWriteDataService {
+    String convertDataToFile(Map<String, Integer> fruitStorage);
```

Comment:
There is no File here. We are getting report as String. So, simple getReport() looks better)